<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleCompetitionConfigWindow</class>
 <widget class="QDialog" name="SingleCompetitionConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>888</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Konfiguracja pojedynczego konkursu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(236, 236, 236);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_startList" stretch="0,0,0,0,0,0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="text">
            <string>Kolejność listy startowej</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <item>
            <widget class="QComboBox" name="comboBox_classification">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Quicksand Medium</family>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: black;</string>
             </property>
             <item>
              <property name="text">
               <string>Według klasyfikacji</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_competition">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Quicksand Medium</family>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: black;</string>
             </property>
             <item>
              <property name="text">
               <string>Według konkursu</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_defaultStartListOrder">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border: 1px solid rgb(0, 59, 23);
border-radius: 6px;
color: rgb(0, 30, 80);
	background-color: rgb(215, 215, 215);
padding: 2px;
}
QPushButton:hover{
background-color: rgb(200, 200, 200);
}</string>
         </property>
         <property name="text">
          <string>Domyślna kolejność</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sortStartListRandomly">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border: 1px solid rgb(0, 59, 23);
border-radius: 6px;
color: rgb(0, 30, 80);
	background-color: rgb(240, 240, 240);
padding: 2px;
}
QPushButton:hover{
background-color: rgb(210, 210, 210);
}</string>
         </property>
         <property name="text">
          <string>Losowa kolejność</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sortStartListByCountries">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border: 1px solid rgb(0, 59, 23);
border-radius: 6px;
color: rgb(0, 30, 80);
	background-color: rgb(240, 240, 240);
padding: 2px;
}
QPushButton:hover{
background-color: rgb(210, 210, 210);
}</string>
         </property>
         <property name="text">
          <string>Kolejność według krajów</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_orderByJumpersList">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border: 1px solid rgb(0, 59, 23);
border-radius: 6px;
color: rgb(0, 30, 80);
	background-color: rgb(240, 240, 240);
padding: 2px;
}
QPushButton:hover{
background-color: rgb(210, 210, 210);
}</string>
         </property>
         <property name="text">
          <string>Kolejność według listy zawodników</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_jumpersLists">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border: 1px solid rgb(0, 59, 23);
border-radius: 6px;
color: rgb(0, 30, 80);
	background-color: rgb(240, 240, 240);
padding: 2px;
}
QPushButton:hover{
background-color: rgb(210, 210, 210);
}</string>
         </property>
         <property name="text">
          <string>Listy zawodników</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_byOtherComp">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border: 1px solid rgb(0, 59, 23);
border-radius: 6px;
color: rgb(0, 30, 80);
	background-color: rgb(240, 240, 240);
padding: 2px;
}
QPushButton:hover{
background-color: rgb(210, 210, 210);
}</string>
         </property>
         <property name="text">
          <string>Lista startowa wg innego konkursu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_loadJumpers">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
color: black;
 background-color: rgb(220, 240, 255); 
border: 1px solid rgb(0, 59, 23);
border-radius: 6px;
 padding: 2px;
} 
QPushButton:hover
{
background-color: rgb(200, 220, 255);
}</string>
         </property>
         <property name="text">
          <string>Wczytaj zawodników z innego pliku</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Quicksand Medium</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>665</width>
          <height>798</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Skocznia</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_hillEditorWidget" stretch="0">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <family>Quicksand Medium</family>
                 <pointsize>12</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: black;</string>
               </property>
               <property name="text">
                <string>Istniejąca skocznia:</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_existingHill">
               <property name="minimumSize">
                <size>
                 <width>280</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>280</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QComboBox{
color: black;
}</string>
               </property>
               <property name="maxVisibleItems">
                <number>25</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>665</width>
          <height>798</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Ustawienia generatora wiatru</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>47</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_competitionOptions">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>11</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="text">
            <string>Belka startowa</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_startGate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(60, 60, 60);</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>-1000</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_autoGate">
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:black;</string>
           </property>
           <property name="text">
            <string>Auto</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>11</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="text">
            <string>DSQ: Jeden skok na</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_dsqProbability">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>148</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>148</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <number>-1</number>
           </property>
           <property name="maximum">
            <number>10000000</number>
           </property>
           <property name="value">
            <number>-1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_jumpsImportance">
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="text">
            <string>Ważność skoków</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_jumpsImportance">
           <property name="minimumSize">
            <size>
             <width>148</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>148</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Quicksand Medium</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_jumpsImportancePreset">
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;</string>
         </property>
         <item>
          <property name="text">
           <string>Preset ważności skoków</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_submit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Quicksand Medium</family>
           <pointsize>23</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border: 1px solid rgb(0, 59, 23);
border-radius: 6px;
color: rgb(244, 244, 244);
background-color:  rgb(5, 80, 116);
}
QPushButton:hover{
background-color: rgb(30, 110, 146);
}</string>
         </property>
         <property name="text">
          <string>Symuluj</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
